stub-idp/src/main/java/uk/gov/ida/stub/idp/saml/transformers/EidasResponseTransformerProvider.java

```diff
@@ -36,13 +37,25 @@ public EidasResponseTransformerProvider(CoreTransformersFactory coreTransformers
         this.digestAlgorithm = digestAlgorithm;
     }
 
+    public Function<Response, String> getTransformer(boolean signAssertions){
+        return StubCoreTransformersFactory.getResponseStringTransformer(
+            encryptionKeyStore,
+            keyStore,
+            entityToEncryptForLocator,
+            signatureAlgorithm,
+            digestAlgorithm,
+            new EncrypterFactory().withDataEncryptionAlgorithm(EncryptionConstants.ALGO_ID_BLOCKCIPHER_AES256_GCM),
```

Comment:
`EncryptionConstants.ALGO_ID_BLOCKCIPHER_AES256` probably can be statically imported.